Cargo.toml

```diff
@@ -503,7 +503,7 @@ required-features = []
 [[example]]
 name = "assets_wasm"
 path = "examples/wasm/assets_wasm.rs"
-required-features = ["bevy_winit"]
+required-features = ["bevy_winit","render"]
```

Comment:
This line is not for the `winit_wasm` example, it's for `assets_wasm`

Could you update the one ten lines down?